woocommerce/woocommerce-gutenberg-products-block
# Move some base components to subfolders

Part of #1962.

This PR moves some base components to subfolders, as discussed in #1962. It doesn't close the issue completely, because there are some other components that we might want to move, but it's a step forward. I didn't move `payment-methods`, for example, to avoid conflicts with ongoing work, but I think it should belong to `cart-checkout` as well.

What is done in this PR:
* `address-form`, `shipping-calculator`, `shipping-location`, `shipping-rates-control` and `totals` are now under `cart-checkout`.
* `checkout` subfolder and `cart-checkout` were merged.
* `review-list`, `review-list-item` and `review-sort-select` are now under `reviews`.

### How to test the changes in this Pull Request:

1. Verify the new folder structure and whether it makes sense.
2. Make sure the project still builds and do some smoke testing to verify _Cart_, _Checkout_ and _Reviews_ blocks continue working.
